gensrc/proto/olap_file.proto

```diff
@@ -56,6 +56,17 @@ enum SegmentsOverlapPB {
     NONOVERLAPPING = 2;
 }
 
+// some txn semantic information bind to a written rowset
+// i.e.
+//   1. partial update txn only provide and update a subset of columns,
+//      so it may add info about updated columns, whether to insert new
+//      rows if old row doesn't exists, and default values etc
+//   2. conditional update txn may add some meta to describe condition
+//   3. more general read-write txn may add complex expressions
+message RowsetTxnMetaPB {
+    repeated string partial_update_columns = 1;
```

Comment:
The use of column names is not a good idea.  It's better to use column_id instead.